### PR TITLE
scala: add page

### PR DESCRIPTION
**Pull request by hohonuuli**
----

- [X] The page (if new), does not already exist in the repo.

- [X] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [X] The page has 8 or fewer examples.

- [X] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [X] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/scala.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# scala
+
+> Scala application launcher and interactive interpreter.
+
+- Call a Scala interactive shell (REPL):
```
Comment (agnivade) [inline pages/common/scala.md]: Perhaps "Launch" is better suited than "Call"

File: pages/common/scala.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+`scala`
+
+- Execute a script in a given Scala file:
```
Comment (agnivade) [inline pages/common/scala.md]: How about "Execute a given Scala script" ?

File: pages/common/scala.md (hunk 1 above)
Comment (hohonuuli) [reply pages/common/scala.md]: "Launch" sounds better to me too. There's no consensus on terms though. [python](https://github.com/hohonuuli/tldr/blob/master/pages/common/python.md) uses "Call", [ruby](https://github.com/hohonuuli/tldr/blob/master/pages/common/ruby.md) uses "Open". The `man scala` page uses "Start". Perhaps we should follow the trend of using a different word for every page ... maybe I should use "Instantiate"? ;-) I'll change it to "Start" to keep it in line with the scala man.

File: pages/common/scala.md (hunk 2 above)
Comment (hohonuuli) [reply pages/common/scala.md]: How about "Execute a Scala script:"

**Comment by hohonuuli** [comment]
👍

**Comment by sbrl** [comment]
Looks ok to me! Thanks, @hohonuuli :D

**Pull request merged**